Python/pylifecycle.c

```diff
@@ -1370,6 +1370,16 @@ Py_FinalizeEx(void)
     runtime->initialized = 0;
     runtime->core_initialized = 0;
 
+    /* Destroy the state of all threads of the interpreter, except of the
+       current thread. In practice, only daemon threads should still be alive,
+       except if wait_for_thread_shutdown() has been cancelled by CTRL+C.
+       Clear frames of other threads to call objects destructors. Destructors
+       will be called in the current Python thread. Since
+       _PyRuntimeState_SetFinalizing() has been called, no other Python thread
+       can take the GIL at this point: if they try, they will exit
+       immediatly. */
```

Comment:
Minor typo:

```suggestion
       immediately. */
```

Also, out of curiosity, regarding what happens if other Python threads attempt to require the GIL after `_PyRuntimeState_SetFinalizing()` is called: could you elaborate a bit on what happens? Do they simply exit silently or with some form of specific error?

(The above question is for furthering my own understanding, not really a specific suggestion for the comment.)

Reply:
Exit silently: see my second PR #18854.